webextensions/README.md

```diff
@@ -0,0 +1,36 @@
+# browser-compat-data
+
+This folder contains compatibility data for the [WebExtensions](https://developer.mozilla.org/Add-ons/WebExtensions) APIs.
+
+It's a JSON file, structured as follows:
+
+* one top level JSON object, which has one property for each WebExtensions API. The name of the property is the name of the module (for example, `"i18n"`, or `"downloads"`). The value is an `api` object.
```

Comment:
> Maybe we should put this under a top-level "apis" property, so we can add additional metadata.

Yeah, this is a good idea for sure. I'd like to add data for manifest keys in here too, which will require this.

> It might also be nice to add a minor and major schema version,
> for backward compatible/non-backward-compatible changes
> we may wind up needing to make.

Yes, I should probably do this. But at the moment I see this as being quite experimental, and wouldn't like to make any compat promises.